### DISCUSSION TITLE
Additional positioning to window Size

### DISCUSSION THREAD

**Discussion by CookieLoid** [f1multiviewer/issue-tracker#347] [Feature requests]
For now the main preset for the places of the windows are 9, but if you have a smaller window size, there are more possibilities.
Possible add if the window is smaller/bigger add more positions like top 1/5, 2/5 etc...

I hope I explained it well, but now you need to manually place them well

**Comment by jeppeolesen** [comment]
Marking this as a duplicate of #20 to keep layout preset discussions in one place
